websites/E/Exaroton/presence.ts

```diff
@@ -0,0 +1,70 @@
+const presence = new Presence({
+    clientId: "792696743686307861"
+  }),
+  presenceData: PresenceData = {
+    largeImageKey: "logo"
+  },
+  paths = {
+    ":": "Home Page",
+    go: "Login Page",
+    account: "Account Settings",
+    server: "Panel - Server",
+    console: "Panel - Console",
+    log: "Panel - Log",
+    options: "Panel - Options",
+    software: "Panel - Software",
+    players: "Panel - Players",
+    files: "Panel - Files",
+    addons: "Panel - Plugins",
+    worlds: "Panel - Worlds",
+    backups: "Panel - Backups",
+    access: "Panel - Access"
+  };
+
+presence.on("UpdateData", async () => {
+  if (document.location.hostname === "exaroton.com") {
+    presenceData.startTimestamp = Date.now();
+    let path = document.location.pathname.replace(
+      /\//g,
+      ""
+    ) as keyof typeof paths;
+    const matchedPath = path.match(/:|software|players|addons/);
+    if (matchedPath && !matchedPath.index)
+      path = matchedPath[0] as keyof typeof paths;
+    const page = paths[path];
+    if (page) presenceData.details = page;
+  } else {
+    const page = document.location.hostname.split(".")[0];
+    presenceData.startTimestamp = Date.now();
+    switch (page) {
+      case "support":
```

Comment:
maybe do
```default:``` ? because like you only switch and check if it is support so maybe if page will not be found, just say idk, watching support or something 🤔

Reply:
Well there isn't any possibility of this happening with Exaroton (or I might be unaware), but I'll just add a `default:` case so that in such a case, it will display **"Browsing..."** under the presence details.

Reply:
ye ye, that's what i wanted thanks :D

Reply:
also add ``break;`` bc u forgot

Reply:
oh yes sorry